hedera-node/src/main/java/com/hedera/services/ledger/EntityChangeSet.java

```diff
@@ -63,6 +63,10 @@ public Map<P, Object> changes(final int i) {
 		return changes.get(i);
 	}
 
+	public void updateChange(final int num, final Map<P, Object> mutableChange) {
```

Comment:
Once we've ensured all `changes` instances of `Map` are mutable, I don't think this should be necessary.